src/services/fields.js

```diff
@@ -0,0 +1,88 @@
+const option = require('option');
+
+class Form {
+  constructor(fields = []) {
+    this.fields = fields;
+  }
+
+  parse(req) {
```

Comment:
perhaps a small comment above each function saying what its for i.e. parse for post requests, store transform to store in session etc.

Reply:
Yep, will do